accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,149 @@
+
+* [Phillip Carter](https://github.com/cartermp): [Add Target Packs on NuGet](https://github.com/dotnet/core/pull/64)
+* [Daniel Plaisted](https://github.com/dsplaisted): [Reference Assembly NuGet packages](https://gist.github.com/dsplaisted/83d67bbcff9ec1d0aff1bea1bf4ad79a)
+* [Prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages)
+
+## Scenarios and User Experience
+
+* Build a project which is multi-targeted to .NET Core or .NET Standard as well
+  as .NET Framework on Mac OS or Linux using the .NET CLI
+
+* Build a project targeting Mono (which uses the same Target Framework
+  Identifier as .NET Framework) using the .NET CLI
+
+## Requirements
+
+### Goals
+
+* Doesn't require running an installer
+* Doesn't require Windows
+* Can be used in SDK-style multi-targeting projects
+
+### Non-Goals
+
+* Supporting NuGet-based acquisition of the .NET Framework runtime
+* Supporting NuGet-based acquisition from non-SDK-style projects
+* Support pre-.NET Framework 4.5 targeting packs
+* Support non-.NET Framework targeting packs (Xamarin, Mono, Unity)
+
+## Design
+
+* Daniel has created a [prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages).
+    - We have an identity package
+    - Each version has a different identity and is looped in by TFM
+    - Ensures only incremental download
+    - Open question is whether we need the aggregate package at all or just
+      compute the package ID from the TFM, but that's not what we did for
+      `Microsoft.NetCore.App` or `NETStandard.Library`. What was the reason?
+
+* Includes .targets files
+    - Locate the files.
+    - Design time?
+
+### Details
+
+* There will be separate NuGet packages with reference assemblies for each
+  version of .NET Framework. This means that projects targeting a single version
+  of .NET Framework don't need to download and spend disk space on the reference
+  assemblies and intellisense files for all the other possible versions of .NET
+  Framework. It also means that when a new version of .NET Framework is
+  released, the reference assemblies for the previous versions don't need to be
+  re-shipped in an updated package along with the new assemblies.
+* There will be a single "metapackage" that can be referenced, that will have
+  conditional dependencies on each of the version-specific reference assembly
+  packages.
+* The metapackage will automatically be referenced by the .NET SDK when required
+  (ie on non-Windows OS's). On Windows, we will probably make the .NET SDK
+  continue to rely on the reference assemblies from the targeting packs by
+  default.
+* For the package IDs, I suggest the following:
+  - For the metapackage: `Microsoft.NETFramework.ReferenceAssemblies`
+  - For the version-specific packages:
+    `Microsoft.NETFramework.ReferencesAssemblies.net462` (where net462 is
+    replaced with the corresponding NuGet short framework Identifier)
+* The NuGet packages will include a `.targets` file that sets the
```

Comment:
I think we may need to set FrameworkPathOverride as well to avoid other weird issues like the explicit mscorlib reference.